### PR TITLE
Update joomla-field-subform.w-c.es6.js

### PR DESCRIPTION
**Pull request by brianteeman**
Spotted a few typos etc in the comments.

Code review only required

### CODE REVIEW

File: build/media_source/system/js/fields/joomla-field-subform.w-c.es6.js [hunk 1]
```diff
@@ -243,7 +243,7 @@
     }
 
     /**
-     * Fix names ind id`s for field that in the row
+     * Fix names and id`s for field that are in the row
```
Comment (richard67) [inline build/media_source/system/js/fields/joomla-field-subform.w-c.es6.js]: „field that are“? Not „fields that are“ or „field that is“?

File: build/media_source/system/js/fields/joomla-field-subform.w-c.es6.js [hunk 2]
```diff
@@ -94,7 +94,7 @@
               : event.target.closest(that.buttonRemove);
           }
 
-          // Check actine, with extra check for nested joomla-field-subform
+          // Check active, with extra check for nested joomla-field-subform
```
Comment (Fedik) [inline build/media_source/system/js/fields/joomla-field-subform.w-c.es6.js:97]: It is French https://fr.wikipedia.org/wiki/Actine multi-functional proteins that form microfilaments.
This need for the subform to be functional.

Just joking, thanks for fixing ;)

File: build/media_source/system/js/fields/joomla-field-subform.w-c.es6.js (hunk 1 above)
Comment (brianteeman) [reply build/media_source/system/js/fields/joomla-field-subform.w-c.es6.js]: i spotted ind which is why I started the PR. I missed the other errors. Thanks for spotting.

**Comment by richard67** [comment]
I have tested this item :white_check_mark: successfully on e97dfd64bcdcc363b551a3a6fedf90dd5abf8ad4<br /><br />Code review.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/32647">issues.joomla.org/tracker/joomla-cms/32647</a>.</sub>

**Comment by Fedik** [comment]
I have tested this item :white_check_mark: successfully on e97dfd64bcdcc363b551a3a6fedf90dd5abf8ad4<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/32647">issues.joomla.org/tracker/joomla-cms/32647</a>.</sub>

**Comment by richard67** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/32647">issues.joomla.org/tracker/joomla-cms/32647</a>.</sub>

**Comment by brianteeman** [comment]
Thanks Robert

**Pull request merged**